### ISSUE TITLE
No error for double methods

### ISSUE THREAD

**Issue by eddy147** [AtomLinter/linter-php#140]
The linter does not give an error if I copy a method in a class.
So, let's say I have a class with a double function like this:

```
<?php
namespace Test;

class A
{
    public function foo()
    {
        
    }

    public function foo()
    {
        
    }
}

```

does not throw an error, but php -l does.

Other stuff, like errors within a function (missing semi-colon, etc, are correctly picked up as errors, so the linter is working.

Atom-editor version: 1.5.3
linter-php version: 1.1.8

**Comment by Arcanemagus** [comment]
What version of PHP are you using?

**Comment by Arcanemagus** [comment]
Nvm, I'm able to reproduce with PHP 5 and PHP 7, investigating.

**Comment by Arcanemagus** [comment]
Ah ha! This is the first _Fatal_ error I've seen, didn't know it was even possible for that output to exist, will be updating the regex to account for that and adding specs. Thanks for reporting this!

**Comment by Arcanemagus** [comment]
Published in v1.2.0 :tada:

**Comment by eddy147** [comment]
Did not expect to get a fix *this quick*.
Great job, it works. Thank you!

**Comment by Arcanemagus** [comment]
The annoying part is when I first took this over I tried finding documentation on the output of that (there is none...), so I tried looking at the PHP source and I _thought_ I read it enough to say that it only ever put out `Parse` errors, but obviously I missed something.
